src/components/NftPreview/NftPreview.tsx

```diff
@@ -162,11 +115,13 @@ function NftPreview({ galleryNftRef }: Props) {
   );
 }
 
+// const StyledLink = styled(Link)``;
+
 const StyledA = styled.a`
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 100%;
+  width: fit-content;
```

Comment:
oh man, let's make sure this doesn't break SVGs haha

Reply:
ah yes good call had on my todo to check and then forgot